index.js

```diff
@@ -70,283 +70,273 @@ Fs.readdirSync(Path.join(__dirname, 'helpers')).forEach(function (file) {
 * @param {assemblerGetTemplates} assembler.getTemplates - Method to assemble templates
 * @param {assemblerGetTranslations} assembler.getTranslations - Method to assemble translations
 */
-function Paper(settings, themeSettings, assembler) {
-    var self = this;
+class Paper {
+    constructor(settings, themeSettings, assembler) {
+        this.handlebars = Handlebars.create();
 
-    self.handlebars = Handlebars.create();
+        this.handlebars.templates = {};
+        this.translator = null;
+        this.inject = {};
+        this.decorators = [];
 
-    self.handlebars.templates = {};
-    self.translator = null;
-    self.inject = {};
-    self.decorators = [];
+        this.settings = settings || {};
+        this.themeSettings = themeSettings || {};
+        this.assembler = assembler || {};
+        this.renderingContext = {};
```

Comment:
Can you name this differently, to be specific to content blocks? Like maybe `contentBlocksContext`